### ISSUE TITLE
Role with implied hidden content has keyboard focus - Section display

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5336]
Sites identified: MCO, Archivo
Page(s) identified: Media object page, section display

WCAG compliance level: A

Description:

Some elements have a role that implies their internal structure should be hidden from assistive technologies. In other words, their sub-elements have a presentational role.

If a sub-element has keyboard focus, it should be moved outside of the presentation role.

Screen readers will ignore any content with a presentation role. If this includes focusable sub-elements, then these will not be announced to the user who might miss out on important information.

Learn more in [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=38100866150&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r90/failed/WkdFdk1xZS8=;NGE3MDcyNzU=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaDlnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f6f1c0bf4fc4de6919ce643f8fac769f386cbfc9/image.png)

<img width="1162" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXZOQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6ca20a89ebc4b41d773ebe0f78d36d2f17f65c8e/image.png" alt="image.png" />

#5564 suggests using a 'list' role for the structured navigation, rather than the 'structure' role in Ramp. Would that work here as well?

# Done Looks Like
- [ ] Change role for section display in current MCO / IJCCR / Archivo instances so that it is not presentational and buttons for expand all / collapse all and expand / collapse of individual sections get keyboard focus

**Comment by elynema** [comment]
As sections will be displayed by Ramp, we should check this in Ramp and then re-create in Ramp repository if it's an issue there.

**Comment by elynema** [comment]
Since remediation is due for MCO, IJCCR, and Archivo in early February, and 7.7 will not roll out until spring break at the absolute earliest, Jon D. would like us to see if we can fix this in place with reasonable effort.

**Comment by elynema** [comment]
Example where this occurs: https://media.dlib.indiana.edu/media_objects/xd07h8220. With the SiteImprove browser plugin on that page, you can see that there are issues for all the buttons that are in the section listing (add to playlist and expand/collapse).

**Comment by cjcolvar** [comment]
I'm working on a commit directly to mco-staging for both this issue and #5570.  It will also include #5341 once a new 1.2 version of iiif-react-media-player gets released.

**Comment by elynema** [comment]
This looks ok on mco-staging. SiteImprove browser plugin identifies this issue on MCO, but not on same page in mco-staging. Looks solution was to change the role to "list" from "tablist"

**Comment by joncameron** [comment]
This looks good to me on mco-staging. I used the SiteImprove browser plugin and the "Role with implied hidden content" issue didn't appear, and "role=list" appeared on the .panel-group element.
